examples/spinning_cube/src/lib.rs

```diff
@@ -1,68 +1,43 @@
 #[macro_use]
 extern crate gdnative as godot;
 
-use godot::init::{Property, PropertyHint, PropertyUsage};
-use godot::GodotString;
-
+#[derive(godot::NativeClass)]
+#[inherit(godot::MeshInstance)]
+#[user_data(godot::user_data::MutexData<RustTest>)]
+#[register_with(my_register_function)]
 struct RustTest {
     start: godot::Vector3,
     time: f32,
+    #[property(default=0.05)]
```

Comment:
```suggestion
    #[property(default = 0.05)]
```

CI is expecting spaces around the `=`.